tests/boardtest.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Just glancing at the script, it doesn't seem like it's using any bash extensions. Any reason why you picked bash and not the standard shell?